CMakeLists.txt

```diff
@@ -171,9 +176,13 @@ configure_package_config_file(cmake/${PROJECT_NAME}Config.cmake.in
 
 if(NOT SKBUILD)
   # -- fastscapelib header files
-  install(FILES ${FASTSCAPELIB_HEADERS}
-          DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/fastscapelib)
-
+  foreach (file ${FASTSCAPELIB_HEADERS})
```

Comment:
Looks like not many people (including me) installed and used the C++ headers so far :).

This is rather critical so I also fixed that in #159 and I'm currently drafting a release 0.2.2.